src/routes/fields/[id]/index.svelte

```diff
@@ -71,10 +71,62 @@
 	};
 </script>
 
+
 <script lang="ts">
+	import BackArrow from '$lib/icons/BackArrow.svelte';
+	import NewIcon from '$lib/icons/NewIcon.svelte';
+  import Modal from '$lib/components/Modal.svelte';
+
+  import Header from '$lib/components/Header.svelte';
+
 	export let field: Field;
+
+
+	function handleBack() {
+		goto('/fields');
+	}
+
+  let showEditModal: boolean = false;
+  const toggleEditModal: () => void = () => {
+    showEditModal = !showEditModal;
+  }
+
+  const handleEditFormSubmit = () => {
+
+  }
 </script>
 
+<Header handleButtonClick={toggleEditModal} />
+
+{#if showEditModal}
```

Comment:
Let's just use the router and have an edit screen (/fields/[id]/edit). Modals are a bit weird on mobile. When we work on the desktop UI, it might make sense to include some.

Reply:
Can look at /varieties/[id]/edit as an example.